### DISCUSSION TITLE
solder reed-switch

### DISCUSSION THREAD

**Discussion by steuerzentrale** [PricelessToolkit/MailBoxGuard#28] [Q&A] [answered]
just a basic question about connecting a reed-switch. where do the two wires of the reed-switch go to, green and yellow, green and blue or something else? its not clear from the documentation and I dont want to break anything ;) maybe you could update that in the documentation or I just didnt find it.

<img width="513" alt="Bildschirmfoto 2023-11-21 um 21 50 06" src="https://github.com/PricelessToolkit/MailBoxGuard/assets/75168550/c89d4fad-e712-4ab6-9e05-19b3609550df">

**Comment by PricelessToolkit** [comment, marked as answer]
"Green, Yellow" = +
"Blue" = Signal input
*********************************************
Green - Blue = onboard sw
Yellow-Blue = external sw
